pkg/networkmanager/firewall-client.es6

```diff
@@ -66,14 +66,6 @@ function fetchServiceInfos(services) {
                     return info;
                 });
     }));
-
-    /*
-     * Work around `cockpit.all()` returning results in individual arguments -
-     * that's just confusing and doesn't work with ES6 style functions.
-     */
-    return promises.then(function () {
-        return Array.prototype.slice.call(arguments);
-    });
```

Comment:
This isn't safe yet, I'm afraid. E. g. on rhel-7-5 we'll have to get along with an older cockpit-system, thus everything that is in Extras can't rely on that fixed API yet.